components/esp_websocket_client/include/esp_websocket_client.h

```diff
@@ -15,6 +15,7 @@
 #include "esp_err.h"
 #include "esp_event.h"
 #include <sys/socket.h>
+#include "esp_transport_ws.h"
```

Comment:
This brings a public dependency to ws-transport.

Reply:
Sorry, does it need any changes or can the conversation be marked as resolved?